http/engine_test.go

```diff
@@ -440,6 +451,21 @@ func TestDecodeURIPath(t *testing.T) {
 	})
 }
 
+func assertServerStarted(t *testing.T, address string) {
```

Comment:
require instead of assert?

Reply:
We use `assertXYZ` all over the place, so kinda trivial in this case I think